Source/missiles.cpp

```diff
@@ -3891,8 +3889,9 @@ void MI_HorkSpawn(Sint32 i)
 					dp = dPiece[tx][ty];
 					if (!nSolidTable[dp] && dMonster[tx][ty] == 0 && dPlayer[tx][ty] == 0 && dObject[tx][ty] == 0) {
 						j = 6;
-						int mon = AddMonster(tx, ty, missile[i]._miVar1, 1, true);
-						M_StartStand(mon, missile[i]._miVar1);
+						direction md = (direction)missile[i]._miVar1;

```

Comment:
Should here also static_casted used?
```suggestion
						direction md = static_cast<direction>(missile[i]._miVar1);

```